retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc

```diff
@@ -0,0 +1,447 @@
+| Object_Type
+| Picklist
+| -
+| N
+| N
+| -
+
+| Managed External Identifier
+| Status
+| Status
+| Picklist
+| -
+| N
+| N
+| -
+
+| Managed External Identifier
+| Status Last Changed
+| Status_Last_Changed
+| Date/Time
+| -
+| N
+| N
+| -
+
+| Order
+| Global_SalesOrder_Id
+| Global_SalesOrder_Id
+| Text
+| 36
+| Y
+| Y
+| -
+
+| Order
+| Total Tax
+| Total_Tax__c
+| Currency
+| -
+| -
+| -
+| -
+
+| Order
+| Shipping
+| Shipping__c
+| Currency
+| -
+| -
+| -
+| -
+
+| Order
+| Order Total Amount
+| OrderTotalAmount__c
+| Formula (Currency)
+| -
+| -
+| -
+| Shipping__c + Total_Tax__c + TotalAmount
+
+| Product
+| Global_Product_Id
+| Global_Product_Id
+| Text
+| 36
+| Y
+| Y
+| -
+
+| Product
+| Product Group
+| Product_Catalog__c
+| Picklist
+| -
+| N
+| N
+| -
+
+| Product
+| Short Description
+| Short_Description__c
+| Text
+| 255
+| N 
+| N
+| -
+
+| Product
+| Item Short Description
+| Item_Short_Description__c
+| Text
+| 255
+| N 
+| N
+| -
+
+| Product
+| Item Long Description
+| Item_Long_Description__c
+| Text
+| 255
+| N 
+| N
+| -
+
+| Product
+| Status
+| Status__c
+| Picklist
+| -
+| N
+| N
+| -
+
+|===
+
+[NOTE]
+Unique fields should be case-sensitive.
+
+=== Create via Apex Script
+
+An Apex class called `CustomFieldUtility` is used to create custom fields programmatically. The source for this class can be found in the https://anypoint.mulesoft.com/exchange/2cc3c2c9-ddd3-4582-bdcc-b510f30065a7/rcg-common-resources/[RCG Common Resources] project. Download this project and follow the steps below.
+
+. Click on the Salesforce Setup icon and select `Developer Console` to open a new console window.
+. Go to `File > New > Apex class` and create a new class named `CustomFieldUtility`.
+. Copy the code from `rcg-common-resources/salesforce/CustomFieldUtility.apxc` to the script editor.
+. Select the `File > Save` menu option to compile and save the class.
+. Select `Debug > Open Execute Anonymous Window`.
+. Copy and paste the contents of `rcg-common-resources/salesforce/CreateCustomFields.txt` into the window.
+. Enable the `Open Log` option and click the *Execute* button.
+
+Once the script completes, select the `Debug Only` filter option to show just the results of each create request. Verify that all fields have been successfully created.
+
+=== Create Manually via Salesforce
+
+To create each of these custom fields and enable visibility for them on page layouts:
+
+. Click on the Salesforce Setup icon.
+. Select the `Object Manager` tab from the Setup home page.
+. Find and select the target Salesforce object.
+. Select the `Fields & Relationships` page.
+. Click the *New* button and create the field as specified above.
+. Once the field has been created, click the `Set Field-Level Security` button (or do these steps later - see below).
+. Enable for desired profile, or tick the checkbox next to `Visible` to enable visibility for the desired profile(s).
+
+Repeat these steps for each custom field in the above table.
+
+[NOTE]
```

Comment:
```suggestion
[NOTE]
====
```

Reply:
I think adding the equal signs will fix this note, but let's test it in a local build before publishing.